faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -455,9 +477,18 @@ private HttpRequest makeHttpRequest(String httpMethod, String path, Optional<Jso
           X_FAUNADB_API_VERSION, API_VERSION,
           USER_AGENT, userAgent,
           X_FAUNA_DRIVER, jvmDriver.toString(),
+          X_FAUNA_DRIVER_VERSION, Package.getPackage("com.faunadb.common").getSpecificationVersion(),
```

Comment:
I believe this API is deprecated.

Reply:
according to doc
```
Deprecated. 
If multiple class loaders delegate to each other and define classes with the same package name, and one such loader relies on the lookup behavior of getPackage to return a Package from a parent loader, then the properties exposed by the Package may not be as expected in the rest of the program.
```
calling Class.class.getPackage() resolve those issue.